model/flow/event.go

```diff
@@ -45,6 +45,15 @@ func (e Event) ID() Identifier {
 	return MakeID(e.Body())
 }
 
+// ByteSize return size of the event in terms of number of bytes
```

Comment:
It's not entirely clear where this size comes from. I assume its because of this:
```go
func EventToMessage(e flow.Event) *entities.Event {
	return &entities.Event{
		Type:             string(e.Type),
		TransactionId:    e.TransactionID[:],
		TransactionIndex: e.TransactionIndex,
		EventIndex:       e.EventIndex,
		Payload:          e.Payload,
	}
}
```

Reply:
yeah, one way can be just counting payload size, as the rest is always fixed and user has no control over :-?

Reply:
I changed it to only measure the payload size